app/views/controllers/account/login/new.html.erb

```diff
@@ -1,29 +1,23 @@
 <%
 add_page_title(:login_please_login.t)
-%>
-
-<%= form_with(scope: :user, url: account_login_path,
-              id: "account_login_form") do |f| %>
-
-  <%= text_field_with_label(form: f, field: :login, value: @login,
-                            label: :login_user.t + ":", class: "mt-3",
-                            data: { autofocus: @login.blank? }) %>
-
-  <%= password_field_with_label(form: f, field: :password, value: "",
-                                label: :login_password.t + ":", class: "mt-3",
-                                data: { autofocus: !@login.blank? }) %>
 
-  <%= check_box_with_label(form: f, field: :remember_me, checked: @remember,
-                           class: "mt-3", label: :login_remember_me.t) %>
+# Simple form object for login
+form_object = Struct.new(:login, :password, :remember_me, keyword_init: true) do
```

Comment:
Not a fan of significantly redundant `Struct.new` calls.  Seems like these could be better handled with a PORO.

Reply:
Agree, poro added (also for add user to group form)